app/components/finance/statements/ecf_statement_selector.rb

```diff
@@ -31,6 +31,10 @@ def t(key)
 
     private
 
+      def cohorts
```

Comment:
I think we should [avoid allowing components to query the database](https://viewcomponent.org/viewcomponents-at-github.html#avoid-database-queries) and instead pass the values in.

Reply:
these are passed in now